src/sentry/backup/helpers.py

```diff
@@ -1,8 +1,11 @@
-from __future__ import annotations
+# Please do not use
+#     from __future__ import annotations
+# in modules such as this one where hybrid cloud data models or service classes are
```

Comment:
This comment could be updated since this file isn't unique to hybrid cloud things.

Reply:
Removed.